### ISSUE TITLE
Termux Not Working

### ISSUE THREAD

**Issue by peru194** [UsergeTeam/Userge#517]
Tried To Deploy In Termux But It Is Telling That Downgrade Python. Plz Fix Termux

**Comment by rking32** [comment]
Yes, known issue. Better to make Userge compatible with python 3.10 instead of finding a workaround for downgrading the python version in Termux.

**Comment by peru194** [comment]
how to make Userge compatible with python 3.10

**Comment by rking32** [comment]
Maybe we have to do some tests with 3.10 and make changes according to it

**Comment by peru194** [comment]
ok plz do that

**Comment by tathastu871** [comment]
tried 3.11 and 3.10.1 and 3.9 but same error

**Comment by tathastu871** [comment]
Solution
apt install tur-repo -y
apy update -y
apt install python* -y
pip3.10 install -r requirements.txt
this willsetup 3.10 and dependencies
